src/components/map/WorldMap.tsx

```diff
@@ -123,14 +123,15 @@ export default function MapChart() {
   const [mapCenter, setMapCenter] = useState<TPoint>([0, 0]);
   const [mapZoom, setMapZoom] = useState(1);
   const [showUnifiedEU, setShowUnifiedEU] = useState(false);
-  const [selectedDocType, setSelectedDocType] = useState<"lawsPolicies" | "unfccc">("lawsPolicies");
+  const [selectedFamCategory, setSelectedFamCategory] = useState<"lawsPolicies" | "unfccc" | "mcf">("lawsPolicies");
```

Comment:
For my clarity - these are corpusTypes (with some gnarly MCF hacking in there?

Reply:
these are not corpus types, but family categories - the latter of which we want to remove later down the line in favour of 'genre'.

Reply:
see here: https://github.com/climatepolicyradar/navigator-db-client/blob/500b4f81bb3eeee4111a80e2e33b1e6068eb28fe/db_client/models/dfce/family.py#L19